x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/user/SystemUser.java

```diff
@@ -36,7 +36,7 @@ public class SystemUser extends InternalUser {
         new RoleDescriptor.IndicesPrivileges[] {
             RoleDescriptor.IndicesPrivileges.builder()
                 .indices("*")
-                .privileges("cross_cluster_replication", "cross_cluster_replication_internal")
+                .privileges("read_cross_cluster", "cross_cluster_replication", "cross_cluster_replication_internal")
```

Comment:
I'm not sure we actually want this here yet, but the test fails with a privilege issue without this since the system user as some point performs the `"internal:data/read/esql/exchange"` action (or one of the other "internal" actions covered by `read_cross_cluster`